### DISCUSSION TITLE
problems with adaptive dashboard

### DISCUSSION THREAD

**Discussion by kasper-leFevre** [UI-Lovelace-Minimalist/UI#953] [General]
Anyone else how have had problems with the initial setup of adaptive dashboard?

I have done everything from the installation guide but nothing is showed when im opening the dahsboard. i can se the navigation point at left menu, and I can see the tabs but the initial template "dummy" dashboard is simply not showing, nothing to see in the console log, and cant see any error in any logs.

**Comment by basbruss** [comment]
What version of `state-switch` are you using? This should be not higher than `v1.9.3`.

**Reply by kasper-leFevre** [reply]
I'm using `v1.9.3`
